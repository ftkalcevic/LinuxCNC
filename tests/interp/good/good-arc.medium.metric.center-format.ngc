g21
f1
g0x0y0z0
g2 x2000 i[0.9995 * 1000]
m2
