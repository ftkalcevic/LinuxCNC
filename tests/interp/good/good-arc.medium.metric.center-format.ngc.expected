    1 N..... USE_LENGTH_UNITS(CANON_UNITS_MM)
    2 N..... SET_G5X_OFFSET(1, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000)
    3 N..... SET_G92_OFFSET(0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000)
    4 N..... SET_XY_ROTATION(0.0000)
    5 N..... SET_FEED_REFERENCE(CANON_XYZ)
    6 N..... USE_LENGTH_UNITS(CANON_UNITS_MM)
    7 N..... SET_FEED_RATE(1.0000)
    8 N..... STRAIGHT_TRAVERSE(0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000)
    9 N..... ARC_FEED(2000.0000, 0.0000, 999.5000, 0.0000, -1, 0.0000, 0.0000, 0.0000, 0.0000)
   10 N..... SET_G5X_OFFSET(1, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000)
   11 N..... SET_XY_ROTATION(0.0000)
   12 N..... SET_FEED_MODE(0, 0)
   13 N..... SET_FEED_RATE(0.0000)
   14 N..... STOP_SPINDLE_TURNING(0)
   15 N..... SET_SPINDLE_MODE(0 0.0000)
   16 N..... PROGRAM_END()
